src/rgw/rgw_sal_motr.cc

```diff
@@ -1021,12 +1021,12 @@ int MotrBucket::link_user(const DoutPrefixProvider* dpp, User* new_user, optiona
 
 }
 
-int MotrBucket::unlink_user(const DoutPrefixProvider* dpp, User* new_user, optional_yield y)
+int MotrBucket::unlink_user(const DoutPrefixProvider* dpp, const rgw_user &new_user, optional_yield y)
```

Comment:
Suggestion:
rename variable new_user  -> bucket_owner

Reply:
+1

Reply:
renamed new_user to bucket_owner